### DISCUSSION TITLE
Let the Deluge shine in terms of generative possibilities :)

### DISCUSSION THREAD

**Discussion by saynotohugs** [SynthstromAudible/DelugeFirmware#1421] [Ideas]
It would be very interesting to incorporate the possibilities from the squarp hapax/pyramid, the OXI one and the OP-Z + more in an intuitive way. Since I'm not a developer I cannot add lines of code to all your great work on the Deluge. 

But with that said I'm an experienced composer with a strong focus on generative sequencing, polymeter/polyrhythms and tonal concepts from modern classical music.  

Besides that I'm trained in UX/digital innovation/design thinking which means that I can develop the basic UI concept in cooperation with developers and test it out with relevant musicians in the process using the revelevant design principles to make sure it's intuitive but also that it's doable by the developers. 

If you could use my help, I'd be very much interested in contributing :)

**Comment by patriciachadwick62** [comment]
tHINK
SHIFT + KEYBOARD MODE
would be a nice place for a GENERATIVE UI

how would you lay what usercontrol ?

**Comment by saynotohugs** [comment]
Well I haven't thought that out and would definately need the feedback from others - and to test it out on users to see if it's intuitive :) 

In my mind it's related to the new draw automation feature. 

Once the correct shortcut is chosen, one way could be to use vertical row 1 for lets say a repeat value, where the upper left pad is value 1, next is value 2 and so forth. This would allow for 16 available parameters on 1 screen with the possibility to add another screen 2 later if necessary. The OLED/7SEQ display could show "repeat 1", "repeat 2" as you press the different pads.

**Comment by mhampton** [comment]
I am wondering how hard it would be to add the same sort of probability options to the grid mode that already exist in clip mode for notes - e.g. you could press a grid point and SELECT and turn to set the probability for that clip to play within a cycle.
